script/setup_local_dev.sh

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+
+: "${ENSURE_K3D:="false"}" # will ensure k3d is installed if set to "true"
+: "${ENSURE_KUBECTL:="false"}" # will ensure kubectl is installed if set to "true"
+: "${ENSURE_HELM:="false"}" # will ensure helm is installed if set to "true"
+
+: "${K3D_VERSION:="v4.4.8"}" # version of k3d to install, empty value uses latest one available. currently pinned to latest known working (due to unresolved port mapping issue: https://github.com/k3d-io/k3d/issues/380)
+: "${K3D_INSTALL_URL:="https://raw.githubusercontent.com/rancher/k3d/main/install.sh"}" # URL to k3d installer script
+: "${K3S_VERSION:="v1.21.1-k3s1"}" # version of k3s to run in k3d cluster, empty value uses default specified by k3d install
+: "${K3D_CLUSTER_NAME:="flyte"}" # name of k3d cluster to be used
+: "${K3D_KUBECONFIG_FILE_PATH:="${HOME}/.k3d/kubeconfig-${K3D_CLUSTER_NAME}.yaml"}" # file path to store kubeconfig file for k3d cluster at
+
+: "${KUBECTL_VERSION:=""}" # version of kubectl to install, empty value uses latest available
+: "${KUBECTL_INSTALL_URL:="https://dl.k8s.io/release/VERSION/bin/linux/amd64/kubectl"}" # URL to kubectl binary, include VERSION to be replaced with KUBECTL_VERSION
+: "${KUBECTL_INSTALL_DIR:="/usr/local/bin"}" # local path to install kubectl to
+: "${KUBECTL_VERIFY_SIGNATURE:="true"}" # verifies signature of kubectl binary downloaded if set to "true"
+
+: "${HELM_VERSION:=""}" # version of helm to install, empty value uses latest v3 available
+: "${HELM_INSTALL_URL:="https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3"}" # URL to helm installer script
+: "${HELM_K8S_NAMESPACE:="flyte"}" # k8s namespace to use with helm
+
+: "${FLYTE_TIMEOUT:=600}"
+
+set -e
+
+HAS_CURL="$(type "curl" &> /dev/null && echo true || echo false)"
+HAS_WGET="$(type "curl" &> /dev/null && echo true || echo false)"
```

Comment:
looks like this checks for `curl` rather than `wget`

Reply:
Oh, whoops :upside_down_face: copy paste strikes again